bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -166,6 +166,11 @@ trait ItcTable extends Table {
 
   }
 
+  // lacking a simple way to decide on the "closest" (i.e. "fastest to reach") peer we talk to the observing peer
+  // if defined and GN or GS otherwise, if no site is defined you're out of luck and the ITC tables will stay empty
+  protected def itcServicePeer: Option[Peer] =
+    ObservingPeer.get orElse ObservingPeer.peerFor(Site.GN) orElse ObservingPeer.peerFor(Site.GS) orElse None
```

Comment:
This seems good to me though ending with `orElse None` is superfluous. That's what you get anyway if none of the others were defined. Should be harmless though. I'll go ahead and merge this one so it can be included in the test release today.